example/src/scenarios/get-crendential.ts

```diff
@@ -146,5 +151,9 @@ export default (
     } catch (e) {
       console.error(e);
       return error(e);
+    } finally {
+      // Clean up ephemeral keys
+      deleteKeyIfExists(WIA_KEYTAG);
+      deleteKeyIfExists(DPOP_KEYTAG);
```

Comment:
Why delete these keys? The Wallet Attestation key must be kept for the entire duration of the Wallet Attestation in case resubmission is necessary.

Reply:
We are regenerating a new wallet instance for each operation here. We used a random uuid before this change so that was basically lost anyway after the flow is done.

Reply:
Ok I would add a comment to say that in the production implementation it should not be eliminated

Reply:
Done in 2a50d9b5a4b22c37128ef4ad32f8c1d8362c34dd, thanks.